### DISCUSSION TITLE
Latest Changes: Better under "Explore Repositories"

### DISCUSSION THREAD

**Discussion by doc-code-hub** [community/community#16589] [Other Feature Feedback, Questions, & Ideas]
Honestly I prefer the only 3 project results of the "Explore Repositories" section over the new Feed (For you)...

This is to say that I didn't like seeing it lowered by the new section "Latest Changes".

![image](https://user-images.githubusercontent.com/61033694/168430125-165b282d-3f18-4c7e-bcfd-c74a11158f96.png)

I would prefer the new "Latest Changes" section to be found below.

---

I'm used to looking in that area to discover some new repository.  
This way you tripped my habit by showing me something I don't really care about (or that interests me secondarily to repos) :)

**Comment by coosir** [comment]
me too
